### ISSUE TITLE
pdal merge runs in standard mode

### ISSUE THREAD

**Issue by abellgithub** [PDAL/PDAL#2961]
It would be better if it could run in stream mode.

**Comment by hobu** [comment]
Wasn't this done?

**Comment by abellgithub** [comment]
No.

**Comment by wrenoud** [comment]
I think this duplicates #2016

**Comment by craigds** [comment]
to confirm, I found this in `pdal --drivers --showjson`:

```
{
        "description": "Merge data from two different readers into a single stream.",
        "extensions": [],
        "link": "https://pdal.io/en/master/stages/filters.merge.html",
        "name": "filters.merge",
        "streamable": true
    }
```

(why isn't this information in the docs for each filter?)

I can't find _where_ in git history this was fixed, but it seems this and #2016 can probably be closed now
